<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMaximaStaticIntegrateAndDiffInterface</class>
 <widget class="QWidget" name="QtMaximaStaticIntegrateAndDiffInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="outputLayout"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="plainTextCheckBox">
         <property name="text">
          <string>Plain text</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Respect to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="respectEdit">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="diffButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Diff</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="integrateButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Integrate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="formulaEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
